src/patternfly/components/Pagination/docs/code.md

```diff
@@ -20,6 +20,7 @@ Note: `<button>` or `<a>` elements can be used in `.pf-c-pagination__nav-page-se
 | Class | Applied To | Outcome |
 | -- | -- | -- |
 | `.pf-c-pagination` | `<div>` |  Initiates pagination. |
+| `.pf-c-pagination__items` | `<div>` | Initiates pagination items. |
```

Comment:
If you didn't change this element's name, would it still be a breaking change?

Reply:
I meant to remove this class.